renovation_core_graphql/reports/sdls/root.graphql

```diff
@@ -0,0 +1,25 @@
+scalar RenovationReportFilterInput
+scalar ReportRowResult
+
+type GetReportResult{
```

Comment:
@e-lobo Let's rename this to `ReportResult`

Reply:
@assemmarwan refactored